lib/components/enter-a-domain-component.js

```diff
@@ -0,0 +1,21 @@
+import { By } from 'selenium-webdriver';
+
+import BaseContainer from '../base-container.js';
+import * as driverHelper from '../driver-helper';
+
+export default class EnterADomainComponent extends BaseContainer {
+	constructor( driver ) {
+		super( driver, By.css( '.map-domain-step__add-domain' ) );
+	}
+
+	enterADomain() {
+		const blogName = 'myawesomedomain.com';
```

Comment:
I think this const needs to be removed and added to the `enterADomain() `line as a variable like `enterADomain( blogName )`

Reply:
Thanks! I should have known this myself. Fixed.